setup/gcp/scripts/run_forseti.sh

```diff
@@ -15,6 +15,8 @@
 
 source /home/ubuntu/forseti_env.sh
 
+set -x
```

Comment:
Please add a comment here that because of this setting we should not put anything private/secret in the commands called here because they will be logged.

Reply:
Comment added, does it look ok to you?